modules/web-console/e2e/testcafe/components/pageAdvancedConfiguration.js

```diff
@@ -23,5 +23,7 @@ export const pageAdvancedConfiguration = {
     cachesNavButton: Selector('.pca-menu-link[ui-sref="base.configuration.edit.advanced.caches"]'),
     async save() {
         await t.click(this.saveButton);
-    }
+    },
+    fieldErrorNotification: Selector('.form-field__error'),
```

Comment:
1. Represent page model as tree of `FormField`s. See pageConfigurationAdvancedGridGain in GG for examples.
2. Each `FormField` already has an error selector property.

Reply:
If you need to select legacy popover validation messages, these are only present on SQL config screen. Select for these would be better placed in SQL config page model because of it's isolated nature.